app/__init__.py

```diff
@@ -38,6 +38,7 @@
 load_dotenv()
 
 db: RoutingSQLAlchemy = RoutingSQLAlchemy()
+setattr(db, "SensitiveString", SensitiveString)
```

Comment:
Can't we just create the actual attribute in our RoutingSQLAlchemy class instead of monkey patching here?

Reply:
Huh, that's a good idea! Probably, let me see...

Reply:
yep, that works!